src/Packlist/PacklistForm.js

```diff
@@ -143,7 +157,7 @@ export default function PacklistForm({ onPacklistSave }) {
   )
 
   function addItem(item) {
-    if (item.text?.trim().length >= 3 && item.text?.trim().length <= 20) {
+    if (item.item?.trim().length >= 3 && item.item?.trim().length <= 20) {
```

Comment:
"item.item" sounds a little bit weird. I'm not sure waht this might be. Maybe you can find some other name for this property that is more descriptive?

Reply:
its the item of the item ;D I dont know, how to name it otherwise.. because it's not the text I need here, so I ha to use item propertie of the item, Maybe I will find another word for the item